Build failure

I'm getting the following build error on the git master branch (currently at commit c5bd97d4):

```
[100%] Built target iHD_drv_video_OBJ
Scanning dependencies of target iHD_drv_video_STATIC
Scanning dependencies of target iHD_drv_video
[100%] Linking CXX static library iHD_drv_video.a
[100%] Linking CXX shared library iHD_drv_video.so
[100%] Built target iHD_drv_video_STATIC
[100%] Built target iHD_drv_video
Scanning dependencies of target RunULT
Running devult...
./devult: symbol lookup error: ../libdrm_mock/libdrm_mock.so: undefined symbol: mos_gem_bo_is_softpin
make[2]: *** [media_driver/linux/ult/ult_app/CMakeFiles/RunULT.dir/build.make:82: RunULT] Error 127
make[1]: *** [CMakeFiles/Makefile2:373: media_driver/linux/ult/ult_app/CMakeFiles/RunULT.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
```

A git bisect shows that this is caused by commit ed04dbcc1d45df4ae88caa8c5565fb3864c1f167.

System information:
- OS: Arch Linux x86_64
- Compiler: gcc 10.2.1
- gmmlib: git master
- libva: git master

--- a/media_driver/linux/ult/libdrm_mock/mos_bufmgr_mock.c
+++ b/media_driver/linux/ult/libdrm_mock/mos_bufmgr_mock.c
@@ -4358,3 +4358,14 @@ fini:
         free(balancer);
     return ret;
 }
+
+bool mos_gem_bo_is_softpin(struct mos_linux_bo *bo)
+{
+    struct mos_bo_gem *bo_gem = (struct mos_bo_gem *) bo;
+    if (bo_gem == nullptr)
+    {
+        return false;
+    }
+
+    return bo_gem->is_softpin;
+}

